models/issues/issue_list.go

```diff
@@ -234,6 +235,12 @@ func (issues IssueList) loadAssignees(ctx context.Context) error {
 
 	assignees := make(map[int64][]*user_model.User, len(issues))
 	issueIDs := issues.getIssueIDs()
+
+	orderBy := "name"
+	if setting.UI.DefaultShowFullName {
+		orderBy = "full_name"
+	}
```

Comment:
I think this sould be a fuction within `user_model "code.gitea.io/gitea/models/user"`

might call it `func GetOrderBy() string` ? or `NameOrderBy` ?

Reply:
Fix in 2da0afa05852b0cc4a7e9e74cbe380ff8c36a589